### PR TITLE
New Site Created w/Persona Bar results in Broken UX Manager

### LINKED ISSUE #332: New Site Created w/Persona Bar results in Broken UX Manager
This is likely due to missing Vanjaro Theme in portal folder. Current workaround is to create a new site using Vanjaro Menu > Tools > Sites

### PR DESCRIPTION
**Pull request by hshimanshusharma**
Close #332

### CODE REVIEW

File: Portals/_default/Skins/Vanjaro/Base.ascx.cs [hunk 1]
```diff
@@ -844,6 +845,22 @@ private void LoadModuleInFrame()
...
+        {
+            try
+            {
+                string ThemeName = Core.Managers.ThemeManager.GetCurrent(PortalSettings.Current.PortalId).Name;
+                string BaseEditorFolder = HttpContext.Current.Server.MapPath("~/Portals/" + PortalSettings.Current.PortalId + "/vThemes/" + ThemeName + "/editor");
+                string ThemeCss = HttpContext.Current.Server.MapPath("~/Portals/" + PortalSettings.Current.PortalId + "/vThemes/" + ThemeName + "/Theme.css");
+                if (!File.Exists(ThemeCss) && !Directory.Exists(BaseEditorFolder))
+                {
+                    ThemeManager.ProcessScss(PortalSettings.Current.PortalId, false);
+                    Config.Touch();
```
Comment (meetmandeep) [inline Portals/_default/Skins/Vanjaro/Base.ascx.cs]: Why Config.Touch()? This restarts site and should not be a requirement to create a new site.

**Review by meetmandeep — CHANGES_REQUESTED** [review]

File: Portals/_default/Skins/Vanjaro/Base.ascx.cs (hunk 1 above)
Comment (hshimanshusharma) [reply Portals/_default/Skins/Vanjaro/Base.ascx.cs]: I have checked uxmode breaking when I will not do restart the application also tried with clear cache both server and client-side but not fixed with both

**Review by meetmandeep — APPROVED** [review]

File: Portals/_default/Skins/Vanjaro/Base.ascx.cs [hunk 2]
```diff
@@ -17,6 +18,7 @@
 using System.Dynamic;
 using System.IO;
 using System.Linq;
+using System.Reflection;
```
Comment (meetmandeep) [inline Portals/_default/Skins/Vanjaro/Base.ascx.cs]: Why do we need reflection?

**Review by meetmandeep — CHANGES_REQUESTED** [review]

File: Portals/_default/Skins/Vanjaro/Base.ascx.cs (hunk 2 above)
Comment (hshimanshusharma) [reply Portals/_default/Skins/Vanjaro/Base.ascx.cs]: Its added by mistake and removed  from the code

**Review by meetmandeep — APPROVED** [review]

**Pull request merged**